plugins/modules/openvswitch_bond.py

```diff
@@ -0,0 +1,386 @@
+version_added: '1.0.6'
+options:
+  bridge:
+    required: true
+    description:
+    - Name of bridge to manage
+    type: str
+  port:
+    required: true
+    description:
+    - Name of port to manage on the bridge
+    type: str
+  interfaces:
+    required: true
+    description:
+    - List of interfaces to add to the bond
+    type: list
+  bond_mode:
+    choices: [ active-backup, balance-tcp, balance-slb ]
+    description:
+    - Sets the bond mode
+    type: str
+  lacp:
+    choices: [ 'active', 'passive', 'off' ]
+    description:
+    - Sets LACP mode
+    type: str
+  bond_updelay:
+    description:
+    - Number of milliseconds a link must be up to be activated
+      to prevent flapping.
+    type: int
+  bond_downdelay:
+    description:
+    - Number of milliseconds a link must be down to be deactivated
+      to prevent flapping.
+    type: int
+  state:
+    default: 'present'
+    choices: [ 'present', 'absent' ]
+    description:
+    - Whether the port should exist
+    type: str
+  timeout:
+    default: 5
+    description:
+    - How long to wait for ovs-vswitchd to respond in seconds
+    type: int
+  external_ids:
+    default: {}
+    description:
+    - Dictionary of external_ids applied to a port.
+    type: dict
+  other_config:
+    default: {}
+    description:
+    - Dictionary of other_config applied to a port.
+    type: dict
+  set:
+    description:
+    - Sets one or more properties on a port.
+    type: list
+"""
+
+EXAMPLES = """
+- name: Create an active-backup bond using eth4 and eth5 on bridge br-ex
+  openvswitch.openvswitch.openvswitch_bond:
+    bridge: br-ex
+    port: bond1
+    interfaces:
+      - eth4
+      - eth5
+    state: present
+- name: Delete the bond from bridge br-ex
+  openvswitch.openvswitch.openvswitch_bond:
+    bridge: br-ex
+    port: bond1
+    state: absent
+- name: Create an active LACP bond using eth4 and eth5 on bridge br-ex
+  openvswitch.openvswitch.openvswitch_bond:
+    bridge: br-ex
+    port: bond1
+    interfaces:
+      - eth4
+      - eth5
+    lacp: active
+    state: present
+# NOTE: other_config values of integer type must be represented
+# as literal strings
+- name: Configure bond with miimon link monitoring at 100 millisecond intervals
+  openvswitch.openvswitch.openvswitch_bond:
+    bridge: br-ex
+    port: bond1
+    interfaces:
+      - eth4
+      - eth5
+    bond_updelay: 100
+    bond_downdelay: 100
+    state: present
+  args:
+    other_config:
+      bond-detect-mode: miimon
+      bond-miimon-interval: '"100"'
+- name: Create an active LACP bond using DPDK interfaces
+  openvswitch.openvswitch.openvswitch_bond:
+    bridge: br-provider
+    port: dpdkbond
+    interfaces:
+      - "0000:04:00.0"
+      - "0000:04:00.1"
+    lacp: active
+    set:
+      - "interface 0000:04:00.0 type=dpdk options:dpdk-devargs=0000:04:00.0"
+      - "interface 0000:04:00.1 type=dpdk options:dpdk-devargs=0000:04:00.1"
+    state: present
+"""
+
+RETURN = """ # """
```

Comment:
Please add details about the returned `commands` list

Reply:
Thanks. Should be addressed now.